src/main/java/com/amihaiemil/docker/ExecInstance.java

```diff
@@ -0,0 +1,25 @@
+package com.amihaiemil.docker;
+
+import java.io.IOException;
+import javax.json.JsonObject;
+
+/**
+ * Exec Api
+ * @version 1.00 2019-12-24
+ * @author Jack Pan
+ * */
+public interface ExecInstance {
```

Comment:
@jackpan123 We already have the execs implemented. A Docker exec is represented by the ``Exec`` interface, we don't need this ``ExecInstance``.